fitnesse/resources/bootstrap-plus/js/bootstrap-plus.js

```diff
@@ -585,4 +592,54 @@ function deleteTag(successData, neededValues) {
 }
 /*
     DELETE END | ADD & DELETE TAGS FUNCTIONS END
-*/
\ No newline at end of file
+*/
+
+function getVersionData(callback, url) {
+     // const $ = require('jquery');
+    $.ajax({
+        type: 'GET',
+        url: url,
+        contentType: 'charset=utf-8',
+        success: data => callback(data),
+        error: function (xhr) {
+            console.log('Error code for version checker: ' + xhr.status, xhr);
+        }
+    });
+}
+
+function versionCheck(data) {
+    if (data != undefined) {
+        data.forEach(versionData => {
+            // Replace property 'version' with 'currentVersion' to make al the property names alike
+            if (versionData.hasOwnProperty('version')) {
+                versionData["currentVersion"] = versionData['version'];
+                delete versionData['version'];
+            }
+
+            // Make version status
+            // Clean string so it can be used as an int for version comparison
+            versionData['formatCurrentVersion'] = parseInt(versionData.currentVersion.replace(/\D/g, ''));
+            versionData['formatLatestVersion'] = parseInt(versionData.latest.replace(/\D/g, ''));
+
+            if (versionData.formatCurrentVersion <= versionData.formatLatestVersion) {
+                if (versionData.formatCurrentVersion < versionData.formatLatestVersion) {
+                    versionData['status'] = 'Outdated';
+                } else if (versionData.formatCurrentVersion === versionData.formatLatestVersion) {
+                    versionData['status'] = 'Up-to-date';
+                }
+            } else {
+                //set status text
+                versionData['status'] = 'Ahead';
+            }
+
+            // Place in html
+            $('#versioncheck').append(
+                '<tr class="check">' +
+                '<td><p>' + versionData.artifactid.replace(/\-/g, ' ') + '</p></td>' +
+                '<td><p>' + versionData.currentVersion.replace('-SNAPSHOT', '') + '</p></td>' +
```

Comment:
Waarom vervang je hier -snapshot? Ik zou juist willen zien dat ik snapshotversies gebruik. Ook om verwarring te voorkomen.

Reply:
ik heb dit gedaan omdat ik dacht dat het zo netter was maar als het juist handig is om te weten haal ik hem met gemakkelijk weer weg